### ISSUE TITLE
[BUG] Not working in OSX Ventura 13.5.2 on M1 Pro

### ISSUE THREAD

**Issue by gallacus** [KoenZomers/RingSnapshotDownload#12]
**Describe the bug**
When I attempt to run ./RingSnapshotDownload, no matter what I do with permissions, it returns:
zsh: killed     ./RingSnapshotDownload

**To Reproduce**
1. Modify Settings.json to include RingUsername, RingPassword
2. Open Terminal
3. Change directory to directory where RingSnapshotDownload is
4. run ./RingSnapshotDownload
5. returns: zsh: killed     ./RingSnapshotDownload

**Expected behavior**
Downloads image, or when using -list provides list of Ring devices

**Screenshots**
None

**Versions (please complete the following information):**
- OS: Ventura 13.5.2 on M1 Pro chip
- RingSnapshotDownload: osx-arm64

**Comment by zmre** [comment]
You might want to check your quarantine settings. Try `xattr -d com.apple.quarantine RingSnapshotDownload` and then run again. You might also need to check the System Preferences Security and Privacy settings as that page sometimes has a warning about running things downloaded from the internet and you may have a prompt there to "open anyway" or something like that.

**Comment by zmre** [comment]
I take it back. I'm having the same problem even after removing the quarantine flag.  System logs show: `ASP: Security policy would not allow process: 555555, /Path/To/RingSNapshotDownload` but I'm not getting a popup or seeing a way to run anyway.  I'm guessing it needs to be signed.  I think there's a way for me to just sign locally and I'll try that next.

**Comment by zmre** [comment]
Yeah, running `codesign` on the binary fixed the issue, but you have to have a mac developer account and code signing cert.  Hope this helps.

**Comment by amotivv** [comment]
You can also create a personal cert and give it code signing to get you going on Mac.
